hedyorg/hedy
# 💻 hide all quizes/parsons and show tabs in levels

Fixes #5141 

**Key points**
- a quize/parson is added as a tab in the customize-class page. This makes it closer to how it'd look like for students.
- the current "hide quize/puzzle" switches will behave as "toggle all quizes/puzzles". Why? Consider the following:
  - the teacher decides to hide all quizes. 
  - they are removed from all levels,
  - they can of course still choose to show a quize in certain levels, while keeping the rest hidden.
  - however, what if they (for whatever reason) want to show all quizes but hide one or two?
  - this scenario is likely because until now the "hide quize" behaves as toggling all quizes, so it should still behave similarly.
  - thus it could be considered as resetting all quizes/puzzles.
- the order of puzzle and quiz tabs are always fixed to be always the last two tabs, respectively.

Something went missing when i added the key translations for https://github.com/hedyorg/hedy/pull/5026 (check https://github.com/hedyorg/hedy/pull/5026/commits/6ee2c4f3161c94a8e5c613b595af840569f83a56#diff-852046eb47f474139974edf5e69ed9758f9fde82e1432c49d772b89aef354b37), so I'm adding them here.

**How to test?**

- go the customize-class page
- add/remove a quiz/puzzle
- reset all puzzles/quizes to test that functionality.
   - _note_ that if you disable all quizes and you add one quiz in level 2, for instance, the disable checkbox will be updated since not all quizes are disabled now. (it needs a refresh for now for that to be reflected in the DOM, but this should be dealt with in #4888)
- log in as a student or preview as teacher to see  your changes from the perspective of students.
- verify that everything works as expected.
